src/scripts/client.js

```diff
@@ -45,11 +45,18 @@ const main = async () => {
     // WARNING: It is best practice to use the VoxeetSDK.initializeToken function to initialize the SDK.
     // Please read the documentation at:
     // https://docs.dolby.io/interactivity/docs/initializing
+    
     VoxeetSDK.initialize('customerKey', 'customerSecret');
+    
+    // Show logging in indicator before user logged in
+    loader.style.display = 'block'
```

Comment:
Why not simply replace the text of "name-message" instead of adding another UI element?

Reply:
Thanks for the suggestion. Resolved.